<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StationAttributeInfoWidget</class>
 <widget class="QWidget" name="StationAttributeInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>573</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Station Attribute Information</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>414</width>
           <height>544</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Input Format:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="inputFormatComboBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="inputFormatSwitchButton">
              <property name="text">
               <string>Switch</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QToolButton" name="stationInfoToolButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QToolButton{
height: 32px;
background-color: #f6f6f8;
border: 1px solid #e9ebec;
border-radius: 4px;
color: black;
}

QToolButton::down-arrow {
 image: url(:/icons/Resources/Icons/DownArrow.png);
}
QToolButton::right-arrow{
 image: url(:/icons/Resources/Icons/RightArrow.png);
}</string>
            </property>
            <property name="text">
             <string>Station Info</string>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <property name="popupMode">
             <enum>QToolButton::DelayedPopup</enum>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
            <property name="arrowType">
             <enum>Qt::DownArrow</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="stationInfoWidget" native="true">
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <property name="leftMargin">
              <number>2</number>
             </property>
             <property name="topMargin">
              <number>2</number>
             </property>
             <property name="rightMargin">
              <number>2</number>
             </property>
             <property name="bottomMargin">
              <number>2</number>
             </property>
             <property name="spacing">
              <number>0</number>
             </property>
             <item row="2" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>ECEF-X(m)</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_3">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Station Name</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="stationNameLabel">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Unknow</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>ECEF-Y(m)</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_5">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Station Code</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>ECEF-Z(m)</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="stationCodeLabel">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Unknow</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="ecefXLabel">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLabel" name="ecefYLabel">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLabel" name="ecefZLabel">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="obsInfoToolButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QToolButton{
height: 32px;
background-color: #f6f6f8;
border: 1px solid #e9ebec;
border-radius: 4px;
color: black;
}

QToolButton::down-arrow {
 image: url(:/icons/Resources/Icons/DownArrow.png);
}
QToolButton::right-arrow{
 image: url(:/icons/Resources/Icons/RightArrow.png);
}</string>
            </property>
            <property name="text">
             <string>OBS Info</string>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <property name="popupMode">
             <enum>QToolButton::DelayedPopup</enum>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
            <property name="arrowType">
             <enum>Qt::DownArrow</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="obsInfoWidget" native="true">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <property name="leftMargin">
              <number>2</number>
             </property>
             <property name="topMargin">
              <number>2</number>
             </property>
             <property name="rightMargin">
              <number>2</number>
             </property>
             <property name="bottomMargin">
              <number>2</number>
             </property>
             <property name="spacing">
              <number>0</number>
             </property>
             <item row="0" column="1">
              <widget class="QLabel" name="startObsTimeLabel">
               <property name="text">
                <string>2025-06-06 12:00:00</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>Interval(s)</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>Epoch Number</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_13">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="text">
                <string>Start OBS Time</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>End OBS Time</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_18">
               <property name="text">
                <string>OBS Data Number</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="endObsTimeLabel">
               <property name="text">
                <string>2025-06-06 12:00:00</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="intervalLabel">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLabel" name="epochNumberLabel">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLabel" name="obsDataNumberLabel">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="recevierInfoToolButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QToolButton{
height: 32px;
background-color: #f6f6f8;
border: 1px solid #e9ebec;
border-radius: 4px;
color: black;
}

QToolButton::down-arrow {
 image: url(:/icons/Resources/Icons/DownArrow.png);
}
QToolButton::right-arrow{
 image: url(:/icons/Resources/Icons/RightArrow.png);
}</string>
            </property>
            <property name="text">
             <string>Receiver Info</string>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <property name="popupMode">
             <enum>QToolButton::DelayedPopup</enum>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
            <property name="arrowType">
             <enum>Qt::DownArrow</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="recevierInfoWidget" native="true">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <property name="leftMargin">
              <number>2</number>
             </property>
             <property name="topMargin">
              <number>2</number>
             </property>
             <property name="rightMargin">
              <number>2</number>
             </property>
             <property name="bottomMargin">
              <number>2</number>
             </property>
             <property name="spacing">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_26">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="text">
                <string>Receiver Manufacturer</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_27">
               <property name="text">
                <string>Receiver Type</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="receiverTypeLabel">
               <property name="text">
                <string>2025-06-06 12:00:00</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="recevierVersionLabel">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_25">
               <property name="text">
                <string>Receiver SN</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_24">
               <property name="text">
                <string>Receiver Version</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="recevierManufacturerLabel">
               <property name="text">
                <string>2025-06-06 12:00:00</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLabel" name="recevierSNLabel">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="antennaInfoToolButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QToolButton{
height: 32px;
background-color: #f6f6f8;
border: 1px solid #e9ebec;
border-radius: 4px;
color: black;
}

QToolButton::down-arrow {
 image: url(:/icons/Resources/Icons/DownArrow.png);
}
QToolButton::right-arrow{
 image: url(:/icons/Resources/Icons/RightArrow.png);
}</string>
            </property>
            <property name="text">
             <string>Antenna Info</string>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <property name="popupMode">
             <enum>QToolButton::DelayedPopup</enum>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
            <property name="arrowType">
             <enum>Qt::DownArrow</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="antennaInfoWidget" native="true">
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="5" column="0">
              <widget class="QLabel" name="label_37">
               <property name="text">
                <string>Antenna SN</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_33">
               <property name="text">
                <string>Measurement Method</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_34">
               <property name="text">
                <string>Antenna Manufacturer</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_35">
               <property name="text">
                <string>Antenna Type</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="measuredAntennaHeightLabel">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_28">
               <property name="text">
                <string>Measured Antenna Height (m)</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_36">
               <property name="text">
                <string>TypeAntennaEINEX</string>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="label_38">
               <property name="text">
                <string>Antenna Height Consistency Compensation (m)</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="measurementMethodLabel">
               <property name="text">
                <string>Antenna Base Bottom</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="antennaManufacturerLabel">
               <property name="text">
                <string>Unknow</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLabel" name="antennaTypeLabel">
               <property name="text">
                <string>Unknow</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLabel" name="typeAntennaEINEXLabel">
               <property name="text">
                <string>Unknow</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QLabel" name="antennaSNLabel">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QLabel" name="antHConsistCompLabel">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
